3.assess_generalizability/nbconverted/generalizabilty_eval.py

```diff
@@ -55,9 +55,6 @@
     )
 )
 
-# # Remove rows where Metadata_genotype is "HET"
```

Comment:
should we be worried about this block deleting?

Reply:
Nope! I remember now in a previous discussion that we should apply the model to all cells including HET and then remove them later for evaluation (generating plots, etc.).